src/Main.elm

```diff
@@ -270,10 +289,10 @@ viewSpeaker speaker =
                 , td [] [ text group ]
                 ]
     in
-    row speaker.id speaker.name speaker.group False
-        :: (speaker.replies
-                |> List.map (\r -> row r.id r.name r.group True)
-           )
+        row speaker.id speaker.name speaker.group False
```

Comment:
Do we use different versions of `elm-format`? I use `elm-format-0.18 0.7.0-exp`.

Reply:
```
➜  ~ elm-format --version
Invalid option `--version'

elm-format-0.18 0.6.1-alpha
```
Should I update the version and see if anything changes?

Reply:
Please do :)

Reply:
Am I using an old npm or something? I keep getting that elm-format version. Which npm version do you have?

```
➜  elm-frontend git:(ch-ch-ch-ch-changes) npm --version                                                  
5.6.0                     
➜  elm-frontend git:(ch-ch-ch-ch-changes) npm install -g elm-format                                      
/home/stian/.nvm/versions/node/v9.5.0/bin/elm-format -> /home/stian/.nvm/versions/node/v9.5.0/lib/node_modules/elm-format/bin/elm-format

> elm-format@0.6.1-alpha install /home/stian/.nvm/versions/node/v9.5.0/lib/node_modules/elm-format       
> binwrap-install         

+ elm-format@0.6.1-alpha  
updated 1 package in 5.546s                         
➜  elm-frontend git:(ch-ch-ch-ch-changes) elm-format --version                                           
Invalid option `--version'

elm-format-0.18 0.6.1-alpha
```

Reply:
Aha, this is my bad, I've installed an experimental version of `elm format` and forgot about it. Read about it here: https://github.com/avh4/elm-format/blob/master/Release%20Notes/0.7.0-exp.md.
I like it however, what do you think?

Reply:
What do you think? https://github.com/roisalen/elm-frontend/pull/4/commits/69ca77a7539f046551230b26302162b3fff3234d

I now have elm-format-0.18 0.7.0-exp.